Test fails with -

``
[ERROR] Failed to execute goal org.codehaus.gmaven:groovy-maven-plugin:2.0:execute (default) on project fuse-build-yaml: Execution default of goal org.codehaus.gmaven:groovy-maven-plugin:2.0:execute failed: Could not find matching constructor for: com.redhat.fuse.VersionManipulator(java.lang.Integer) -> [Help 1]
``

This is most likely due to the pipeline hookup for apicurito & quickstarts, the VersionManipulator is expecting a scm tag string like ``spring-boot-camel-xa-1.0.0.fuse-720013`` but instead is getting ``1``

--- a/pom.xml
+++ b/pom.xml
@@ -99,47 +99,47 @@
     <spring-boot-cxf-jaxws.version>1.0.0.fuse-710004</spring-boot-cxf-jaxws.version>
 
     <!-- Tag and SCM info -->
-    <karaf.tag>1</karaf.tag>
+    <karaf.tag>DUMMY-TAG-0.0.0</karaf.tag>
     <karaf.fetch.scmUrl>1</karaf.fetch.scmUrl>
     <karaf.push.scmUrl>1</karaf.push.scmUrl>
     <karaf.buildCommand>1</karaf.buildCommand>
-    <cxf-xjc-utils.tag>1</cxf-xjc-utils.tag>
+    <cxf-xjc-utils.tag>DUMMY-TAG-0.0.0</cxf-xjc-utils.tag>
     <cxf-xjc-utils.fetch.scmUrl>1</cxf-xjc-utils.fetch.scmUrl>
     <cxf-xjc-utils.push.scmUrl>1</cxf-xjc-utils.push.scmUrl>
     <cxf-xjc-utils.buildCommand>1</cxf-xjc-utils.buildCommand>
-    <cxf.tag>1</cxf.tag>
+    <cxf.tag>DUMMY-TAG-0.0.0</cxf.tag>
     <cxf.fetch.scmUrl>1</cxf.fetch.scmUrl>
     <cxf.push.scmUrl>1</cxf.push.scmUrl>
     <cxf.buildCommand>1</cxf.buildCommand>
-    <camel.tag>1</camel.tag>
+    <camel.tag>DUMMY-TAG-0.0.0</camel.tag>
     <camel.fetch.scmUrl>1</camel.fetch.scmUrl>
     <camel.push.scmUrl>1</camel.push.scmUrl>
     <camel.buildCommand>1</camel.buildCommand>
-    <fuse-extras-distro.tag>1</fuse-extras-distro.tag>
+    <fuse-extras-distro.tag>DUMMY-TAG-0.0.0</fuse-extras-distro.tag>
     <fuse-extras-distro.fetch.scmUrl>1</fuse-extras-distro.fetch.scmUrl>
     <fuse-extras-distro.push.scmUrl>1</fuse-extras-distro.push.scmUrl>
     <fuse-extras-distro.buildCommand>1</fuse-extras-distro.buildCommand>
-    <camel-extra.tag>1</camel-extra.tag>
+    <camel-extra.tag>DUMMY-TAG-0.0.0</camel-extra.tag>
     <camel-extra.fetch.scmUrl>1</camel-extra.fetch.scmUrl>
     <camel-extra.push.scmUrl>1</camel-extra.push.scmUrl>
     <camel-extra.buildCommand>1</camel-extra.buildCommand>
-    <wsdl2rest.tag>1</wsdl2rest.tag>
+    <wsdl2rest.tag>DUMMY-TAG-0.0.0</wsdl2rest.tag>
     <wsdl2rest.fetch.scmUrl>1</wsdl2rest.fetch.scmUrl>
     <wsdl2rest.push.scmUrl>1</wsdl2rest.push.scmUrl>
     <wsdl2rest.buildCommand>1</wsdl2rest.buildCommand>
-    <fuse-components.tag>1</fuse-components.tag>
+    <fuse-components.tag>DUMMY-TAG-0.0.0</fuse-components.tag>
     <fuse-components.fetch.scmUrl>1</fuse-components.fetch.scmUrl>
     <fuse-components.push.scmUrl>1</fuse-components.push.scmUrl>
     <fuse-components.buildCommand>1</fuse-components.buildCommand>
-    <hawtio.tag>1</hawtio.tag>
+    <hawtio.tag>DUMMY-TAG-0.0.0</hawtio.tag>
     <hawtio.fetch.scmUrl>1</hawtio.fetch.scmUrl>
     <hawtio.push.scmUrl>1</hawtio.push.scmUrl>
     <hawtio.buildCommand>1</hawtio.buildCommand>
-    <hawtio-online.tag>1</hawtio-online.tag>
+    <hawtio-online.tag>DUMMY-TAG-0.0.0</hawtio-online.tag>
     <hawtio-online.fetch.scmUrl>1</hawtio-online.fetch.scmUrl>
     <hawtio-online.push.scmUrl>1</hawtio-online.push.scmUrl>
     <hawtio-online.buildCommand>1</hawtio-online.buildCommand>
-    <fuse-karaf.tag>1</fuse-karaf.tag>
+    <fuse-karaf.tag>DUMMY-TAG-0.0.0</fuse-karaf.tag>
     <fuse-karaf.fetch.scmUrl>1</fuse-karaf.fetch.scmUrl>
     <fuse-karaf.push.scmUrl>1</fuse-karaf.push.scmUrl>
     <fuse-karaf.buildCommand>1</fuse-karaf.buildCommand>
@@ -147,63 +147,63 @@
     <kubernetes-model.fetch.scmUrl>1</kubernetes-model.fetch.scmUrl>
     <kubernetes-model.push.scmUrl>1</kubernetes-model.push.scmUrl>
     <kubernetes-model.buildCommand>1</kubernetes-model.buildCommand>
-    <kubernetes-client.tag>1</kubernetes-client.tag>
+    <kubernetes-client.tag>DUMMY-TAG-0.0.0</kubernetes-client.tag>
     <kubernetes-client.fetch.scmUrl>1</kubernetes-client.fetch.scmUrl>
     <kubernetes-client.push.scmUrl>1</kubernetes-client.push.scmUrl>
     <kubernetes-client.buildCommand>1</kubernetes-client.buildCommand>
-    <spring-cloud-kubernetes.tag>1</spring-cloud-kubernetes.tag>
+    <spring-cloud-kubernetes.tag>DUMMY-TAG-0.0.0</spring-cloud-kubernetes.tag>
     <spring-cloud-kubernetes.fetch.scmUrl>1</spring-cloud-kubernetes.fetch.scmUrl>
     <spring-cloud-kubernetes.push.scmUrl>1</spring-cloud-kubernetes.push.scmUrl>
     <spring-cloud-kubernetes.buildCommand>1</spring-cloud-kubernetes.buildCommand>
-    <docker-maven-plugin.tag>1</docker-maven-plugin.tag>
+    <docker-maven-plugin.tag>DUMMY-TAG-0.0.0</docker-maven-plugin.tag>
     <docker-maven-plugin.fetch.scmUrl>1</docker-maven-plugin.fetch.scmUrl>
     <docker-maven-plugin.push.scmUrl>1</docker-maven-plugin.push.scmUrl>
     <docker-maven-plugin.buildCommand>1</docker-maven-plugin.buildCommand>
-    <narayana-spring-boot.tag>1</narayana-spring-boot.tag>
+    <narayana-spring-boot.tag>DUMMY-TAG-0.0.0</narayana-spring-boot.tag>
     <narayana-spring-boot.fetch.scmUrl>1</narayana-spring-boot.fetch.scmUrl>
     <narayana-spring-boot.push.scmUrl>1</narayana-spring-boot.push.scmUrl>
     <narayana-spring-boot.buildCommand>1</narayana-spring-boot.buildCommand>
-    <spring-boot-camel-xa.tag>1</spring-boot-camel-xa.tag>
+    <spring-boot-camel-xa.tag>DUMMY-TAG-0.0.0</spring-boot-camel-xa.tag>
     <spring-boot-camel-xa.fetch.scmUrl>1</spring-boot-camel-xa.fetch.scmUrl>
     <spring-boot-camel-xa.push.scmUrl>1</spring-boot-camel-xa.push.scmUrl>
     <spring-boot-camel-xa.buildCommand>1</spring-boot-camel-xa.buildCommand>
-    <fabric8v2.tag>1</fabric8v2.tag>
+    <fabric8v2.tag>DUMMY-TAG-0.0.0</fabric8v2.tag>
     <fabric8v2.fetch.scmUrl>1</fabric8v2.fetch.scmUrl>
     <fabric8v2.push.scmUrl>1</fabric8v2.push.scmUrl>
     <fabric8v2.buildCommand>1</fabric8v2.buildCommand>
-    <fabric8-maven-plugin.tag>1</fabric8-maven-plugin.tag>
+    <fabric8-maven-plugin.tag>DUMMY-TAG-0.0.0</fabric8-maven-plugin.tag>
     <fabric8-maven-plugin.fetch.scmUrl>1</fabric8-maven-plugin.fetch.scmUrl>
     <fabric8-maven-plugin.push.scmUrl>1</fabric8-maven-plugin.push.scmUrl>
     <fabric8-maven-plugin.buildCommand>1</fabric8-maven-plugin.buildCommand>
-    <fuse-patch.tag>1</fuse-patch.tag>
+    <fuse-patch.tag>DUMMY-TAG-0.0.0</fuse-patch.tag>
     <fuse-patch.fetch.scmUrl>1</fuse-patch.fetch.scmUrl>
     <fuse-patch.push.scmUrl>1</fuse-patch.push.scmUrl>
     <fuse-patch.buildCommand>1</fuse-patch.buildCommand>
-    <wildfly-camel.tag>1</wildfly-camel.tag>
+    <wildfly-camel.tag>DUMMY-TAG-0.0.0</wildfly-camel.tag>
     <wildfly-camel.fetch.scmUrl>1</wildfly-camel.fetch.scmUrl>
     <wildfly-camel.push.scmUrl>1</wildfly-camel.push.scmUrl>
     <wildfly-camel.buildCommand>1</wildfly-camel.buildCommand>
-    <apicurito.tag>1</apicurito.tag>
+    <apicurito.tag>DUMMY-TAG-0.0.0</apicurito.tag>
     <apicurito.fetch.scmUrl>1</apicurito.fetch.scmUrl>
     <apicurito.push.scmUrl>1</apicurito.push.scmUrl>
     <apicurito.buildCommand>1</apicurito.buildCommand>
-    <redhat-fuse.tag>1</redhat-fuse.tag>
+    <redhat-fuse.tag>DUMMY-TAG-0.0.0</redhat-fuse.tag>
     <redhat-fuse.fetch.scmUrl>1</redhat-fuse.fetch.scmUrl>
     <redhat-fuse.push.scmUrl>1</redhat-fuse.push.scmUrl>
     <redhat-fuse.buildCommand>1</redhat-fuse.buildCommand>
-    <fuse-apicurito-generator.tag>1</fuse-apicurito-generator.tag>
+    <fuse-apicurito-generator.tag>DUMMY-TAG-0.0.0</fuse-apicurito-generator.tag>
     <fuse-apicurito-generator.fetch.scmUrl>1</fuse-apicurito-generator.fetch.scmUrl>
     <fuse-apicurito-generator.push.scmUrl>1</fuse-apicurito-generator.push.scmUrl>
     <fuse-apicurito-generator.buildCommand>1</fuse-apicurito-generator.buildCommand>
-    <wildfly-camel-examples.tag>1</wildfly-camel-examples.tag>
+    <wildfly-camel-examples.tag>DUMMY-TAG-0.0.0</wildfly-camel-examples.tag>
     <wildfly-camel-examples.fetch.scmUrl>1</wildfly-camel-examples.fetch.scmUrl>
     <wildfly-camel-examples.push.scmUrl>1</wildfly-camel-examples.push.scmUrl>
     <wildfly-camel-examples.buildCommand>1</wildfly-camel-examples.buildCommand>
-    <fuse-eap.tag>1</fuse-eap.tag>
+    <fuse-eap.tag>DUMMY-TAG-0.0.0</fuse-eap.tag>
     <fuse-eap.fetch.scmUrl>1</fuse-eap.fetch.scmUrl>
     <fuse-eap.push.scmUrl>1</fuse-eap.push.scmUrl>
     <fuse-eap.buildCommand>1</fuse-eap.buildCommand>
-    <ipaas-quickstarts.tag>1</ipaas-quickstarts.tag>
+    <ipaas-quickstarts.tag>DUMMY-TAG-0.0.0</ipaas-quickstarts.tag>
     <ipaas-quickstarts.fetch.scmUrl>1</ipaas-quickstarts.fetch.scmUrl>
     <ipaas-quickstarts.push.scmUrl>1</ipaas-quickstarts.push.scmUrl>
     <ipaas-quickstarts.buildCommand>1</ipaas-quickstarts.buildCommand>
@@ -215,11 +215,11 @@
     <application-templates.buildCommand>1</application-templates.buildCommand>
 
     <!-- Syndesis project -->
-    <atlasmap.tag>1</atlasmap.tag>
+    <atlasmap.tag>DUMMY-TAG-0.0.0</atlasmap.tag>
     <atlasmap.fetch.scmUrl>1</atlasmap.fetch.scmUrl>
     <atlasmap.push.scmUrl>1</atlasmap.push.scmUrl>
     <atlasmap.buildCommand>1</atlasmap.buildCommand>
-    <syndesis.tag>1</syndesis.tag>
+    <syndesis.tag>DUMMY-TAG-0.0.0</syndesis.tag>
     <syndesis.fetch.scmUrl>1</syndesis.fetch.scmUrl>
     <syndesis.push.scmUrl>1</syndesis.push.scmUrl>
     <syndesis.buildCommand>1</syndesis.buildCommand>

